.github/workflows/tests.yml

```
@@ -26,6 +26,17 @@ jobs:
         run: |
           echo "${{ secrets.LOCAL_PROPERTIES }}" > local.properties
 
+      - name: Decode keystore
+        if: matrix.os == 'ubuntu-latest'
```

actionlint (1.7.9)

property "os" is not defined in object type {}

(expression)